src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalCompletionItem.ts

```diff
@@ -71,6 +72,17 @@ export interface ITerminalCompletion extends ISimpleCompletion {
 	 * Whether the completion is a keyword.
 	 */
 	isKeyword?: boolean;
+
+	/**
+	 * Unresolved completion item from the language server provider/
+	 */
+	_unresolvedItem?: CompletionItem;
```

Comment:
Trying to add fields here so that we can store unresolved item, so we can later lazily resolve them when that specific completion item is focused.